lib/terraform/group_attributes.rb

```diff
@@ -29,8 +29,8 @@ def self.coerce(config_value)
     end
 
     def resolve(client:)
-      client.each_output_name { |name| soft_store key: name, value: name }
       dup.each_pair { |key, value| store key, client.output(name: value) }
+      client.each_output { |k, v| soft_store key: k, value: v['value'] }
```

Comment:
Yeah, just was thinking changing the order would change what was returned. Don't remember if it matters here or not (seems to still work at least).